packages/twenty-server/src/database/commands/command-runners/active-or-suspended-workspaces-migration.command-runner.ts

```diff
@@ -117,39 +137,53 @@ export abstract class ActiveOrSuspendedWorkspacesMigrationCommandRunner<
       this.logger.log(chalk.yellow('Dry run mode: No changes will be applied'));
     }
 
-    try {
-      for (const [index, workspaceId] of activeWorkspaceIds.entries()) {
-        this.logger.log(
-          `Running command on workspace ${workspaceId} ${index + 1}/${activeWorkspaceIds.length}`,
-        );
+    const appVersion = this.environmentService.get('APP_VERSION');
```

Comment:
I would leave it to the upgrade command to pull the appVersion, does not seem to be a concern of this active-or-suspended command